feat: add worker amount faq (#1088)
shopware/docs: guides/hosting/infrastructure/message-queue.md

* `MESSENGER_TRANSPORT_DSN` - The DSN to the transport to use (e.g. `doctrine://default`).
* `MESSENGER_TRANSPORT_FAILURE_DSN` - The DSN to the transport to use for failed messages (e.g. `doctrine://default?queue_name=failed`).

## How many workers do I need?

> The number of workers depends on the amount of messages that are queued and which type of messages they are. Product indexing messages are usually slow, while other messages are processed very fast. Therefore it's difficult to give a general recommendation. You should monitor the queue and adjust the number of workers accordingly.
> Sometimes it makes also sense to route messages to a different transport to limit the amount of workers for a specific type of message to avoid database locks or prioritize messages like sending emails.

Reviewer: ```suggestion
The number of workers depends on the number of messages queued and the type of messages they are. Product indexing messages are usually slow, while other messages are processed very fast. Therefore, it is difficult to give a general recommendation. You should be able to monitor the queue and adjust the number of workers accordingly.
Sometimes, it also makes sense to route messages to a different transport to limit the number of workers for a specific type of message to avoid database locks or prioritize messages like sending emails.
```
